### DISCUSSION TITLE
Got access to Copilot for CLI, but when authenticating in CLI, it erroneously thinks I don't have a Copilot subscription.

### DISCUSSION THREAD

**Discussion by Abstracting** [community/community#54611] [npm] [answered]
### Select Topic Area

Question

### Body

Got access to Copilot for CLI.
![2023-05-06 10 01 000131](https://user-images.githubusercontent.com/20521408/236612057-536f803a-e263-45db-ad58-36e77b4f3cb8.jpg)

I have a Copilot subscription.
![2023-05-06 09 59 000129](https://user-images.githubusercontent.com/20521408/236612070-c9c32201-29a3-43cb-8f6c-16e8ff794a54.jpg)

I get this error when I try to authenticate:
![2023-05-06 10 00 000130](https://user-images.githubusercontent.com/20521408/236612091-d4dd7ceb-11f6-40b7-b53d-c1a4c7df1953.jpg)
`Failed to authenticate: GitHub Copilot access is required to use this CLI. Please sign up at https://github.com/features/copilot.`

I checked that I'm logged into GitHub before I paste the auth link. It validates the code correctly.

What to do?

**Comment by Akash1134** [comment, marked as answer]
Hi @Abstracting ,

Sorry that Copilot for CLI is not working as expected.

Could you please remove the cached GitHub and Copilot access tokens located in the following location, try re-authenticating and check if it helps?
~/.copilot-cli-access-token 
~/.copilot-cli-copilot-token

Thanks!

**Reply by avanastar** [reply]
this didn't work for me I'm able to use `gh copilot explain "sudo apt-get"` for example but authenticating with `github-copilot-cli auth`

**Comment by Abstracting** [comment]
Slow respons from me. That worked yes. Thanks!

**Reply by Akash1134** [reply]
Awesome! Glad to know that it helped. 🙂

**Comment by teobatman00** [comment]
Please response to accept me, I have sent request for more than 5 months

**Comment by zarifpour** [comment]
Is there still a waitlist?

It says completed on the next website, however I am getting this error with an active copilot subscription:

```
Failed to authenticate: You do not have access to GitHub Copilot CLI. Please join the waitlist at https://githubnext.com/projects/copilot-cli/., Unable to verify waitlist access.
```

**Comment by carsonbird** [comment]
Same issue here, and I've followed the steps above.

**Comment by sh910913** [comment]
I've followed the steps. It still not working.
<img width="1039" alt="image" src="https://github.com/community/community/assets/29350642/7803c8d8-1c4c-48ac-9244-108d858e9629">

**Comment by UseeNadabo** [comment]
Good program

**Comment by jbenettius** [comment]
I get the same error when authenticating. Re-authenticating does not help. I am able to authenticate, with CoPilot, using the "gh" tool.

**Comment by calops** [comment]
Same here, no way to log into this. Documentation is non-existent. Is this project actually released right now?

**Comment by Pabl0Parra** [comment]
Same issue here, I have tried removing the "~/.copilot-cli-access-token" (cannot find the ~/..copilot-cli-copilot-token). Same error, unable to verify waitlist access (which i am on). I have a paid subscription of Copilot and its working. Please help

**Comment by reason42** [comment]
I have the same issue and it is not clear if there is a wait list or not.  I have organisation copilot for github and enabled the beta feature (GitHub Copilot in the CLI) for my team and we all get _Unable to verify waitlist access._

**Reply by reason42** [reply]
I think this is the way:

https://docs.github.com/en/copilot/github-copilot-in-the-cli/using-github-copilot-in-the-cli

**Reply by jonaustin** [reply]
This works completely fine on my Arch Linux box (`github-copilot-cli auth`)

But no matter what I do, it always gives me the waitlist error on Mac OS.

I finally gave up trying to using the npm one and just used the above using-github-copilot-in-the-cli instructions and simply overrode the `??` alias to `alias "??"="gh copilot explain"`
(I _think_ ^ is literally the same thing as github-copilot-cli's ?? alias)

This tool is incredibly cool and useful for helping folks level up in the CLI, but sadly this is one of the most confusing rollouts I've ever seen...
